From 3bd5562fa42d7ff393eccee74d013b97a2edca42 Mon Sep 17 00:00:00 2001
From: Sam Ravnborg <sam@ravnborg.org>
Date: Mon, 15 Jan 2018 21:35:08 +0100
Subject: [PATCH 16/16] arm9_cpu: load barebox.bin

The preferred bootloader is barebox.bin, so search for that file.
It is confusing that we name barebox for u-boot.

Signed-off-by: Sam Ravnborg <sam@ravnborg.org>
---
 contrib/board/skov/arm9_cpu/arm9_cpu_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig b/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig
index 30d87b4..0af138f 100644
--- a/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig
+++ b/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig
@@ -3,3 +3,4 @@ CONFIG_RAM_128MB=y
 CONFIG_SDCARD=y
 CONFIG_AT91_MCI1=y
 CONFIG_JUMP_ADDR="0x23800000"
+CONFIG_IMAGE_NAME="barebox.bin"
-- 
2.12.0
